README.md

```diff
@@ -7,7 +7,7 @@ managers** including but not exhaustive of updating, searching and ordering supp
 via Command Line Interface (CLI).
 
 ## User interface
-![Ui](docs/images/Ui.png)
+![Ui](docs/images/Warenager.png)
```

Comment:
Based on the CS2103T website, the UI mockup should be put on `docs/images/Ui.png` instead of `docs/images/Warenager.png`.